Fluid/Ast/ForStatement.cs

```diff
@@ -46,7 +47,10 @@ public class ForStatement : TagStatement
         public LiteralExpression Offset { get; }
```

Comment:
one thing I noticed is that many types like `LiteralExpression` have not been marked as sealed. if it were sealed, JIT could determine that here it would be direct method call to `LiteralExpression` instead of being prepared it to be a virtual call to some class defined outside of seen scope. so ideally seal when not expected to be extended outside of this library.